galvani/BioLogic.py

```diff
@@ -207,6 +207,7 @@ def MPTfileCSV(file_or_path):
     431: ('|Zwe-ce|/Ohm', '<f4'),
     432: ('Re(Zwe-ce)/Ohm', '<f4'),
     433: ('-Im(Zwe-ce)/Ohm', '<f4'),
+    438: ('step time/s','<f8'),
```

Comment:
I'm trying to keep these in sorted order, so the new line should go between 435 and 441.